### PR TITLE
Scores

### LINKED ISSUE #119: Show score

### PR DESCRIPTION
**Pull request by aggeagge**

### CODE REVIEW

File: src/client/game.ts [hunk 1]
```diff
@@ -12,17 +12,7 @@ import State from '../common/state';
...
-  LOGICAL_TO_PIXELS,
-  PLAYER_SIZE,
-  ENEMY_SIZE,
-  ENEMY_SPRITE,
-  HP_BAR_WIDTH,
-  HP_BAR_HEIGHT,
-  HP_BAR_FLOAT,
-  SERVER_BROADCAST_RATE,
-} from '../common/constants';
+import * as CONSTANTS from '../common/constants';
```
Comment (stan3984) [inline src/client/game.ts]:
```suggestion
import * as constants from '../common/constants';
```

File: src/client/game.ts [hunk 2]
```diff
@@ -409,10 +411,45 @@ export default class ClientGame extends GameLoop {
     this.up = new Key('ArrowUp');
     this.right = new Key('ArrowRight');
     this.down = new Key('ArrowDown');
-    this.fire = new Key(' '); //Spacebar
+    this.fire = new Key(' '); //Spacebar¨
```
Comment (stan3984) [inline src/client/game.ts]: ?

File: src/common/simulation.ts [hunk 3]
```diff
@@ -318,9 +318,12 @@ export default abstract class Simulation {
     this.state.players[player.id].target.x = point.x;
     this.state.players[player.id].target.y = point.y;
     const userData = fixture.getBody().getUserData() as { id: number }; ///to get id of the target
-    this._state.enemies[userData?.id]?.takeDamage(
+    const enemyDead = this._state.enemies[userData?.id]?.takeDamage(
       this.state.players[player.id].weapons[0].attack_damage,
     );
+    if (enemyDead) {
+      this.state.players[player.id].addScore(10);
```
Comment (stan3984) [inline src/common/simulation.ts]: `constants.ts`

**Review by stan3984 — CHANGES_REQUESTED** [review]

**Review by Dojlando — APPROVED** [review]

File: src/client/game.ts (hunk 2 above)
Comment (erikvader) [reply src/client/game.ts]: https://en.wiktionary.org/wiki/%C2%A8

**Review by stan3984 — APPROVED** [review]

**Review by erikvader — APPROVED** [review]

**Pull request merged**